ccpage114.py

amymhaddad opened the pull request:
There are multiple (but short) exercises in this file. I intentionally separate each exercise with a double space and place the instructions as comments.

paulghaddad commented on ccpage114.py:
```diff
@@ -0,0 +1,145 @@
+    'city': 'columbus',
+}
+
+people = [friend_1, friend_2, friend_3]
```
Nice work!

paulghaddad commented on ccpage114.py:
```diff
@@ -0,0 +1,145 @@
+# Next, loop through your list and as you do print everything you know about each pet.
+pets = {
+    'toby': {
+        'animal_type': 'dog',
```
Great variable names.

paulghaddad commented on ccpage114.py:
```diff
@@ -0,0 +1,145 @@
+    
+    'kitty': {
+        'animal_type': 'hampster',
+        'owner_name': 'chris',
```
lol, hilarious.

paulghaddad commented on ccpage114.py:
```diff
@@ -0,0 +1,145 @@
+        },
+}
+
+for pet_name, pet_info in pets.items():
```
Good work unpacking the separate values.

paulghaddad commented on ccpage114.py:
```diff
@@ -0,0 +1,145 @@
+}
+
+for pet_name, pet_info in pets.items():
+    print(f"\nPet name: {pet_name.capitalize()}")
```
I'd recommend adding the line break at the _end_ (line 61), as opposed to the start. More conventional.

paulghaddad commented on ccpage114.py:
```diff
@@ -0,0 +1,145 @@
+# Think of three names to use as keys in the dictionary, and store one to three favorite places for each person.
+# Loop through the dictionary, and print each person’s name and their favorite places.
+favorite_places = {
+    'sal': ['canada', 'ireland'],
```
Good choice of using a list to store the places. Good design.

paulghaddad commented on ccpage114.py:
```diff
@@ -0,0 +1,145 @@
+
+# Modify your program from Exercise 6-2 (page 102) so each person can have more than one favorite number.
+# Then print each person’s name along with their favorite numbers .
+names_and_fav_numbers = {
```
👏

paulghaddad commented on ccpage114.py:
```diff
@@ -0,0 +1,145 @@
+
+# Make a dictionary called cities.
+# Use the names of three cities as keys in your dictionary.
+# Create a dictionary of information about each city and include the country that the city is in, 
```
Style!

paulghaddad commented on ccpage114.py:
```diff
@@ -0,0 +1,145 @@
+    },
+}
+
+for city, city_info in cities.items():
```
Nice variable names.

paulghaddad commented on ccpage114.py:
```diff
@@ -0,0 +1,145 @@
+    """)
+
+
+# Use one of the example pro- grams from this chapter, and extend it by adding new keys and values,
```
I would suggest trying to add another ingredient to the `pizza` dictionary. We can discuss.

paulghaddad commented on ccpage114.py:
```diff
@@ -0,0 +1,145 @@
+}
+
+for city, city_info in cities.items():
+    if city_info['country'] == 'usa':
```
I see what you're doing here. Nice job.

amymhaddad commented:
@paulghaddad Here's what I did:
-Removed trailing white spaces and added spaces where hound suggested
-Added line break per your suggestion at line 63
-Updated last exercise, beginning at line 133. Is this what you had in mind?
